packages/@ama-sdk/core/src/clients/api-fetch-client.ts

```diff
@@ -16,13 +16,19 @@ import {BaseApiClientOptions} from '../fwk/core/base-api-constructor';
 import {CanceledCallError, EmptyResponseError, ResponseJSONParseError} from '../fwk/errors';
 import {ReviverType} from '../fwk/Reviver';
 
-/** @see BaseApiClientOptions */
+/**
+ * @see BaseApiClientOptions
+ * @deprecated Use the one exposed by {@link @ama-sdk/client-fetch}, will be removed in v13
```

Comment:
shouldn't we remove it in v14 ?

Reply:
V12 or v13 but definitively not v14.
Not sure we want to keep dead code for 1 year and half

Reply:
we should probably discuss and update the documentation about breaking changes then https://github.com/AmadeusITGroup/otter/blob/main/CONTRIBUTING.md#rules-for-contributions

Reply:
Indeed the documentation is confusing:

> Note that only even major Otter versions allow breaking changes. This means we only allow breaking changes in n+2 major versions.

Here the second sentence guideline is respected but, even if it has be flagged as consequence of the first one, not the rule on "even major version".

Anyway, as discussed, we will need to adapt this part to make it more flexible